test/token_validation_tests.js

```diff
@@ -68,22 +72,39 @@ describe("Persona Client Test Suite - Token Validation Tests", function() {
             });
         });
 
-        afterEach(function() {
+        afterEach(function afterEachTest() {
+            if (this.nockAssertions) {
+                lodash.forEach(this.nockAssertions, function eachNock(nockAssertion) {
```

Comment:
Better function name might be `verifyEachNockRequest` - it's not clear to someone outside what this is doing.